model/src/main/kotlin/PackageCuration.kt

```diff
@@ -27,6 +27,11 @@ import org.ossreviewtoolkit.utils.ort.showStackTrace
 
 import org.semver4j.Semver
 
+/**
```

Comment:
Looking down the call hierarchy I ended up here:

https://github.com/semver4j/semver4j/blob/3e5362de90b525876753404e61d849636a40120d/src/test/java/org/semver4j/RangesListFactoryTest.java#L38

I now wonder whether `||` and `<=` and so forth need also to be considered.

Reply:
`||` should, yes. `<=` is already covered by `<` and `=`.